android/src/main/java/com/bitmovin/player/reactnative/RNPlayerView.kt

```diff
@@ -270,6 +295,14 @@ class RNPlayerView(
         val aspectRatio =
             player?.playbackVideoData
                 ?.let { Rational(it.width, it.height) }
+                ?.let { rational ->
```

Comment:
fix for https://app.bugsnag.com/angel-studios/angel-studios-mobile/errors/617c5f3c7f841a00074ecf15?filters[event.since]=30d&filters[error.status]=open&filters[search][]=too&filters[search][]=extreme